webapp/src/components/modals/create_issue/create_issue_form.tsx

```diff
@@ -128,14 +128,13 @@ export default class CreateIssueForm extends React.PureComponent<Props, State> {
             if (error) {
                 state.error = error.message;
             } else {
-                const fields = this.filterInvalidFields(this.state.projectKey, this.state.issueType, this.state.fields);
                 state.fields = fields;
```

Comment:
`fields` isn't defined here, so I'm not sure what this line is doing

Reply:
@Kshitij-Katiyar Please take a look when you have the chance

Reply:
@mickmister sorry the late reply removed the statement